pkg/router/routerclient/client_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/SkycoinProject/skywire-mainnet/internal/testhelpers"
 
 	"github.com/SkycoinProject/dmsg/cipher"
+
```

Comment:
Could you please split and sort the imports into three groups? Standard library, packages other than `skywire-mainnet` and `skywire-mainnet`? Thanks!

Reply:
Done